ansible/roles/manage-server-prod/templates/application.yml.j2

```diff
@@ -17,17 +17,26 @@ server:
 features: {{ manage.features }}
 
 push:
-  url: https://{{ engine_api_domain }}/api/connections
-  user: {{ engine_api_metadata_push_user }}
-  password: {{ engine_api_metadata_push_password }}
-  name: {{ instance_name }} EngineBlock
-  exclude_edugain_imports: {{ manage_exclude_edugain_imports_in_push }}
+  eb:
+    url: https://{{ engine_api_domain }}/api/connections
+    user: {{ engine_api_metadata_push_user }}
+    password: {{ engine_api_metadata_push_password }}
+    name: {{ instance_name }} EngineBlock
+    exclude_edugain_imports: {{ manage_exclude_edugain_imports_in_push }}
+    exclude_oidc_rp: {{ manage_exclude_oidc_rp_imports_in_push }}
+  oidc:
+    url: https://oidcng.{{ base_domain }}/manage/connections
+    user: manage
+    name: {{ manage.oidcng_name }}
+    password: {{ oidcng_api_metadata_push_password }}
+    enabled: {{ manage.oidc_push_enabled }}
 
 product:
   name: Manage
   organization: {{ instance_name }}
   service_provider_feed_url: {{ manage_service_provider_feed_url }}
-
+  supported_languages: {{ supported_language_codes }}
+  show_oidc_rp: {{ manage_show_oidc_rp_tab }}
```

Comment:
Again, the configured value for `manage_show_oidc_rp_tab` is False, having it set to be True would be preferable.